ipapython/install/core.py

```diff
@@ -160,7 +159,7 @@ def _knob(type=_missing, default=_missing, bases=_missing, _order=_missing,
 
     if bases is _missing:
         bases = (KnobBase,)
-    elif isinstance(bases, types.TypeType):
+    elif isinstance(bases, type):
```

Comment:
No, it's the same thing since Python 2.2 and has been removed from Python 3:
```
$ python2.7
>>> import types
>>> types.TypeType is type
True
$ python3.5
>>> import types
>>> types.TypeType is type
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: module 'types' has no attribute 'TypeType'

```